### ISSUE TITLE
how to recreate results of CUB200-2011 as mentioned in the paper? what are the hyperparameters needed to recreate the result?

### ISSUE THREAD

**Issue by euwern** [CompVis/metric-learning-divide-and-conquer#6]
python experiment.py --dataset cub --nb-clusters 4 --finetune-epoch 190 

I have tried to recreate the result for CUB200-2011 dataset, under the following hyperparameter, but my best results are NMI=66.91 and R@1=62.01. I wonder under what hyperparameters that can one achieve NMI=69.60 and R@1=65.90? 

Please advise, Thanks

**Comment by asanakoy** [comment]
1. You have to use distance weighted sampling. In this repository we used a simpler sampling scheme.
Also changing dataloaders merging in https://github.com/CompVis/metric-learning-divide-and-conquer/blob/master/lib/data/loader/utils.py#L82 to use minimum of  the batches over all clusters (i.e. `nb_batches = min(nb_batches_per_dl)` may help.
2. Set `finetune-epoch` to something lower like ~60 and decrease the learning rate every 30 epochs for example.
3. Another thing which you can do is increasing the embedding size to 256.
